rotkehlchen/tests/unit/decoders/test_hop.py

```diff
@@ -101,20 +175,519 @@ def test_hop_optimism_eth_receive_no_event(database, optimism_inquirer, optimism
         database=database,
         tx_hash=tx_hash,
     )
+    bridge_amount, timestamp = '0.03958480553397407', TimestampMS(1666977475000)
     user_address = optimism_accounts[0]
     expected_events = [
         EvmEvent(
             tx_hash=tx_hash,
             sequence_index=0,
-            timestamp=TimestampMS(1666977475000),
+            timestamp=timestamp,
             location=Location.OPTIMISM,
             event_type=HistoryEventType.WITHDRAWAL,
             event_subtype=HistoryEventSubType.BRIDGE,
             asset=A_ETH,
-            balance=Balance(amount=FVal('0.03958480553397407')),
+            balance=Balance(amount=FVal(bridge_amount)),
             location_label=user_address,
-            notes='Bridge 0.03958480553397407 ETH via hop protocol',
+            notes=f'Bridge {bridge_amount} ETH via Hop protocol',
             counterparty=CPT_HOP,
-            address=string_to_evm_address('0x86cA30bEF97fB651b8d866D45503684b90cb3312'),
-        )]
+            address=ETH_AMM_WRAPPER_OPTIMISM,
+        ),
+    ]
+    assert events == expected_events
+
+
+@pytest.mark.vcr(ilter_query_parameters=['apikey'])
```

Comment:
You have a typo in the entire file. Wherever you use this. Please be careful. Does not fail, is just ignored.

**Fix in the entire file**

```suggestion
@pytest.mark.vcr(filter_query_parameters=['apikey'])
```